### DISCUSSION TITLE
fail to get applications after upgrading docker image to  477b62acf065

### DISCUSSION THREAD

**Discussion by BonderTam** [logto-io/logto#1865] [Q&A]
![image](https://user-images.githubusercontent.com/249648/188043561-904c22ed-1e2c-48fd-bd72-a2f7f6ab6c83.png)
![image](https://user-images.githubusercontent.com/249648/188043670-b1b621fe-72d7-454e-b8fe-b599312a24ac.png)
![image](https://user-images.githubusercontent.com/249648/188043712-29f291c8-11d3-4d00-80bd-12b866b52c0f.png)

is data schema changed? how to update on Production?

**Comment by BonderTam** [comment]
![image](https://user-images.githubusercontent.com/249648/188044251-22ac5361-776f-4dc2-b12a-0da49d9e4ecf.png)
cannot create application

**Comment by charIeszhao** [comment]
Can you please provide more context? Like:
* What was the version you previously used, and what's the current version deployed?
* Are you using our docker image or docker compose?
* Can you send us the detailed message of the error?
